src/main/java/com/exactpro/th2/common/schema/factory/AbstractCommonFactory.java

```diff
@@ -642,13 +642,12 @@ public EventID getRootEventId() {
                     }
```

Comment:
the getRootEventId method must not return null

Reply:
done